apps/telegram-bot/core/handlers/url_process.py

```
@@ -0,0 +1,225 @@
+            text=f"Processing the {i + 1}th url...",
+        )
+        url_metadata = await api_client.get_url_metadata(url, ban_list=TELEGRAM_BOT_MESSAGE_BAN_LIST)
+        if url_metadata["source"] == "banned":
+            await process_message.edit_text(
+                text=f"For the {i + 1} th url, the url is banned."
+            )
+            return
+        if url_metadata["source"] == "unknown":
+            if GENERAL_SCRAPING_ON:
+                await process_message.edit_text(
+                    text=f"Uncategorized url found. General webpage parser is on, Processing..."
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)